src/pages/auction/PurchaseComponent/PurchaseComponent.tsx

```diff
@@ -82,13 +82,14 @@ const PurchaseComponent: React.FC<PurchaseComponentProps> = ({
     }
 
     const { slots } = store.getState().slots;
-    const slotNames = slots.map(({ name }) => name || '');
-    const {
-      bestMatch: { rating },
-      bestMatchIndex,
-    } = findBestMatch(message, slotNames);
+    const { search } = new Fuse(slots, {
```

Comment:
Why did you choose `fuse.js` over the `string-similarity`? Does it give any benefits?

Reply:
@Kozjar, `string-similarity` is deprecated and not supported.
The library for complex search is good for solving the problem of finding a similar string.
It also allows you to isolate yourself from a particular implementation of the search in favor of using the optimal algorithm chosen by the library.